apps/web-client/src/lib/db/orders/customers.ts

```diff
@@ -27,10 +27,17 @@ export async function upsertCustomer(db: DB, customer: Customer) {
 		]
 	);
 }
+export const getAllCustomerOrderLines = async (db: DB) => {
```

Comment:
I know you're probably still working on write up/docs, but just  reminder to add jsdoc comments with a description for each handler, no matter simplisitic:

/**
  * Retrieves all customer order lines from the db. A customer order line represents a single book in a customer order - the book meta data & timestamps to indicate when/if it has been placed, ordered with the supplier or received
  */